docs/nodes/nodes-library/core-nodes/Compression/README.md

```diff
@@ -0,0 +1,81 @@
+---
+permalink: /nodes/n8n-nodes-base.compression
+description: Learn how to use the Compression node in n8n
+---
+
+# Compression
+
+The Compression node is useful to compress and decompress files. You can either use the `gzip` or `zip` file format.
+
+
+## Node Reference
+
+- Operations
+	- Compress
+	- Decompress
+
+- Options
+    - ***Binary Property*** field: This field allows you to specify the name of the binary property
+    - ***Output Prefix*** field: This field allows you to specify a prefix for the name of the file gets generated
```

Comment:
This is only visible if the output format is set to gzip, that should be specified.